compatibility/core/src/main/java/org/mule/compatibility/core/transport/AbstractConnector.java

```diff
@@ -405,17 +405,18 @@ public final synchronized void start() throws MuleException {
       return;
     }
 
-    scheduler = createScheduler();
-
     if (!isConnected()) {
+      scheduler = createScheduler();
```

Comment:
That scheduler only makes sense to handle situations that happen only when a connection is active. Will rename the variable to reflect this.